apps/davi/src/components/ProposalVotesCard/ProposalVotesCard.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  SidebarCard,
+  SidebarCardContent,
+  SidebarCardHeaderSpaced,
+} from 'components/SidebarCard';
+import { SidebarCardContentWrapper } from 'components/SidebarCard/SidebarCard.styled';
+import { useTranslation } from 'react-i18next';
+import { useHookStoreProvider } from 'stores';
+import { VoteRow } from './components';
+import { VotesAmount, VotesAmountWrapper } from './ProposalVotesCard.styled';
+import { ProposalVotesCardProps } from './types';

```

Comment:
For new components, it would be nice to adhere to ESlint import order:

https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md

Reply:
i think i got it right, let me know otherwise